test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlAttributeTest.cs

```diff
@@ -122,30 +122,6 @@ public void MultiValueExpressionAttribute()
         }
 
         [Fact]
-        public void VirtualPathAttributesWorkWithConditionalAttributes()
-        {
-            ParseBlockTest("<a href='@foo ~/Foo/Bar' />",
```

Comment:
Would prefer we keep one of these, showing the straightforward set of `Span`s and `Block`s generated. Helps to show `~` remains not special at this level.